docs/manage/fleet/provision/setup.md

```diff
@@ -198,16 +200,18 @@ Create a file called <FILE>viam-provisioning.json</FILE> with the following form
 
 ```json {class="line-numbers linkable-line-numbers"}
 {
-  "manufacturer": "Skywalker",
-  "model": "C-3PO",
-  "fragment_id": "2567c87d-7aef-41bc-b82c-d363f9874663",
-  "hotspot_prefix": "skywalker-setup",
-  "disable_captive_portal_redirect": false,
-  "hotspot_password": "skywalker123",
-  "turn_on_hotspot_if_wifi_has_no_internet": false,
-  "offline_before_starting_hotspot_minutes": "3m30s",
-  "user_idle_minutes": "2m30s",
-  "retry_connection_timeout_minutes": "15m"
+  "network_configuration": {
+    "manufacturer": "Skywalker",
+    "model": "C-3PO",
+    "fragment_id": "2567c87d-7aef-41bc-b82c-d363f9874663",
+    "hotspot_prefix": "skywalker-setup",
+    "disable_captive_portal_redirect": false,
+    "hotspot_password": "skywalker123",
+    "turn_on_hotspot_if_wifi_has_no_internet": false,
+    "offline_before_starting_hotspot_minutes": "3m30s",
+    "user_idle_minutes": "2m30s",
+    "retry_connection_timeout_minutes": "15m"
```

Comment:
It's confusing that these are listed in the "example" but not the "template." Shouldn't the template be the exhaustive case? And example be the more minimal set with values actually filled out?